dotCMS/src/integration-test/java/com/liferay/portal/language/LanguageUtilTest.java

```diff
@@ -69,31 +76,61 @@ public void getLanguageId_existing_expected_non_negative_lang_Test() throws DotS
         APILocator.getLanguageAPI().deleteLanguage(language);
     }
 
+    private static class TestCaseLanguageLocale {
+        String localeRequested;
+        long expectedLangId;
+
+        TestCaseLanguageLocale(final String localeRequested,final long expectedLangId){
+            this.localeRequested = localeRequested;
+            this.expectedLangId = expectedLangId;
+        }
+    }
+
+    @DataProvider
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit 4 tests that execute tests should use the @Test annotation, JUnit 5 tests should use @Test, @RepeatedTest, @TestFactory, @TestTemplate or @ParameterizedTest](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5588495)